omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -74,7 +74,15 @@ public String getPersonAttributeDisplayValue(Person person, String attributeType
 		if (person != null) {
 			PersonAttribute attr = person.getAttribute(Context.getPersonService().getPersonAttributeTypeByUuid(attributeTypeUuid));
 			if (attr != null) {
-				return attr.toString();
+				// special case to display Yes/No for Booleans, since the Core PersonAttribute toString does not do this for us
+				if (attr.getHydratedObject() != null && attr.getHydratedObject().getClass().equals(Boolean.class)){
```

Comment:
shouldn't this just be "att.getHydratedObject() instanceof Boolean"?

Reply:
Make sense, will fix.